### PR TITLE
Added Note about timeout issue when connecting to DB

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 10/umbraco-cms/fundamentals/setup/upgrading/version-specific/upgrade-from-8-to-latest.md [hunk 1]
```diff
@@ -54,13 +54,17 @@ If you use Umbraco Forms, make sure to have [`StoreUmbracoFormsInDbset`](https:/
...
-You can also add the connection details if you spin up the clean installation&#x20;
+You can also add the connection details if you spin up a clean installation.
 {% endhint %}
 
 4. Run the new project and login to authorize the upgrade.
 5. Select "Upgrade" when the upgrade wizard appears.
 6. Once the upgrade has been completed, it's recommended to login to the backoffice to verify if your project is upgraded to new version.
 
+{% hint style="warning" %}
+If you experience timeout issues when connecting to a database, set the 'Connection Timeout' value in the **ConnectionStrings** section of the `appsettings.json` file.
```
Comment (alina-tincas) [inline 10/umbraco-cms/fundamentals/setup/upgrading/version-specific/upgrade-from-8-to-latest.md:65]: can you link to an article where one can read more about this and how they can do this more in depth?

**Comment by alina-tincas** [comment]
Hi @eshanrnh thank you for the PR 🙌 

I have given some suggestions, can you please take a look into them? 😊

File: 10/umbraco-cms/fundamentals/setup/upgrading/version-specific/upgrade-from-8-to-latest.md (hunk 1 above)
Comment (eshanrnh) [reply 10/umbraco-cms/fundamentals/setup/upgrading/version-specific/upgrade-from-8-to-latest.md]: Added References.

**Comment by eshanrnh** [comment]
Thanks for your review, @alina-tincas 🙌 I've incorporated the comments.

File: 10/umbraco-cms/fundamentals/setup/upgrading/README.md [hunk 2]
```diff
@@ -88,6 +88,12 @@ If you have other packages installed such as Umbraco Forms, then before upgradin
 
 9. Restart your site in IIS, build, and run your project to finish the installation.
 
+{% hint style="warning" %}
+If your database experiences timeout issues after an upgrade, it might be due to [ASP.NET Core Module's](https://learn.microsoft.com/en-us/aspnet/core/test/troubleshoot-azure-iis?#default-startup-limits) 'startupTimeLimit' configuration.
+
+To fix the issue, try increasing the [`startupTimeLimit`](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/web-config?) in the `web.config` file. Additionally, you can set the [`Connection Timeout`](https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectiontimeout?) value in the [`ConnectionStrings`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnection.connectionstring?) section of the `appsettings.json` file.
```
Comment (alina-tincas) [inline 10/umbraco-cms/fundamentals/setup/upgrading/README.md:94]:
```suggestion
To fix the issue, try increasing the [`startupTimeLimit`](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/web-config?) in the `web.config` file. Additionally, you can set the [`Connection Timeout`](https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectiontimeout?) value in the [`ConnectionString`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnection.connectionstring?) in the `appsettings.json` file.
```

**Review by alina-tincas — APPROVED** [review]
One last suggestion and apart from this all is good 💪

**Pull request merged**